### DISCUSSION TITLE
Wann ist zeitlich eine neue Version >13.0.8 mit allen aktuellen Rollings geplant

### DISCUSSION THREAD

**Discussion by sammykd** [jomjol/AI-on-the-edge-device#1878] [General]
Hallo, es gibt ja mittlerweile eine menge Rollings, wann werden diese in einer offiziellen Version >13.0.8 zur Verfügung gestellt.
Vielleicht liegt es auch an mir, aber ich blicke bei den Rollingversionen mit deren Bezeichnung nicht durch und kann nicht erkennen welche die letzte Rollingversion sein sollte. Für mich stellt sich das immer nur für eine Rollingversion für ein Problem da, sehe ich das richtig oder wie liest man die Rollingversionen von der Historie her? Vielleicht auch weil mir GitHup nicht so geläufig ist.

**Comment by caco3** [comment]
Der `rolling` branch ist der Entwicklungsbranch.
Normalerweise entwickeln wir ein Bugfux oder neues Feature in einem Branch und mergen den dann ins `rolling` rein.
Und bei einem Release mergen wir jeweils `rolling` in `master`.

Bei dem Commit (d.h. Änderung der Dateien, wird automatisch die Firmware und UI neu gebaut und über https://github.com/jomjol/AI-on-the-edge-device/actions verfügbar gemacht.

Das neuste `rolling` kriegst du Jeweils, wenn du das oberste build nimmst, welches mit `rolling` markiert ist.
Oder direkt  über diesen Link: https://github.com/jomjol/AI-on-the-edge-device/actions/workflows/build.yaml?query=branch%3Arolling

Wir sind übrigens momentan intensiv am testen und wenn wir dann das Gefühl haben, dass es stabil ist, wird es nächstes ein release 14 geben.

Wir sind aber dankbar für alle, welche sich ans `rolling` getrauen und uns so Fehler melden können, bevor sie ins Release kommen.

**Reply by sammykd** [reply]
Hallo caco3,
nach deiner Erklärung wird mir das Finden von rolling Versionen klar.
Danke für die Aufklärung

**Reply by ozett** [reply]
good explanantion of the rolling-release,
makes it easier to try it.

should go into the documentation
(as a more geneeral explanation for "living at the edge".

**Reply by caco3** [reply]
Thanks, I updated https://jomjol.github.io/AI-on-the-edge-device-docs/rolling-installation/ a bit

Please feel free to update it yourself, we are happy about any support especially for the documentation!

**Reply by ozett** [reply]
looks cleaner.
the section has a "catchy" but somehow playful wording, which maybe should sound like a well-known song-title https://cutt.ly/L9dB2aK

what about raising support for the development with participation on the rolling release?
if you want to bring that forwared, what about ranmaing the section not so playful to 

"Support our Development - Try out early features and fixes in `Rolling-Relaease`"
"Improve Code-Base - Try out early features and fixes in `Rolling-Relaease`"

**Reply by caco3** [reply]
> Living on the edge

That is a common know phrase and I think it is ok to use it here, see https://www.collinsdictionary.com/de/submission/6227/Living+on+the+edge

We do not want to push users to use `rolling` because we do not want to have support and discussions about it. Whoever uses `rolling` knows what he does and is taking an increased risk.

For that a pre-release or beta release fits better.

**Reply by ozett** [reply]
ok. sounds reasonable.
and if i forgot to mention: This all is a great idea and project, overall!
